frontend/ui/widget/datetimewidget.lua

```diff
@@ -396,6 +399,15 @@ function DateTimeWidget:createLayout()
     self:refocusWidget()
 end
 
+function DateTimeWidget:addWidget(widget)
+    table.insert(self.date_frame[1],  #self.date_frame[1], widget)
+end
+
+function DateTimeWidget:getAddedWidgetAvailableWidth()
+    return self.date_frame[1][1].width - 2*Size.padding.default
+end
```

Comment:
I let @hius07 tell us if this API is similar or not to the existing ones we already have to add other bits/widgets to higher level widgets (I don't really remember which allows that, I think some gets added some radio buttons).
I think we'd want this to be quite standard.

Reply:
Can it be a checkbox in the plugin menu, "Show timer in status bar"?

Reply:
> I think we'd want this to be quite standard.

It is not currently.